Source/Initialization/InjectorMomentum.H

```diff
@@ -153,34 +153,30 @@ struct InjectorMomentumBoltzmann
     getMomentum (amrex::Real const x, amrex::Real const y, amrex::Real const z,
                  amrex::RandomEngine const& engine) const noexcept
     {
+        using namespace amrex::literals;
+
         // Calculate the local temperature and check if it's too
         // high for Boltzmann or less than zero
         amrex::Real const theta = temperature(x,y,z);
-        if (theta < 0) {
+        if (theta < 0._rt) {
             amrex::Abort("Negative temperature parameter theta encountered, which is not allowed");
         }
         // Calculate local velocity and abort if |beta|>=1
         amrex::Real const beta = velocity(x,y,z);
-        if (beta <= -1 || beta >= 1) {
+        if (beta <= -1_rt || beta >= 1_rt) {
```

Comment:
Just adding the `.` for consistency (but I think it's fully equivalent):
```suggestion
        if (beta <= -1._rt || beta >= 1._rt) {
```